Server/Components/Pages/GamePage.razor

```diff
@@ -14,12 +14,16 @@
 		<div class="equipment-flex">
 			EKWIPUNEK
 
+			<ItemCard Item="item"></ItemCard>
```

Comment:
Hm, ta zmiana tutaj powinna być? Ona już jest w głównym kodzie, bo to chyba ja robiłem?